readme.md

```diff
@@ -25,8 +25,16 @@ jss source.css -p > source.jss
 ### API
 
 ```javascript
-import {cssToJss} from 'jss-cli'
-cssToJss(css, options)
+var css = [
+  '.ninja {',
+  '  visibility: hidden;',
+  '  color: black;',
+  '}'
+].join('\n')
+
+var css2jss = require('../lib/cssToJss.js')
+var jss = css2jss(css)
+console.log(JSON.stringify(jss, null, 2))
```

Comment:
yes, i just figured out, that es6 template strings are already in chrome.
ok, so i'll change the example, but in order to run anywhere, i would then also include `babelify`.